taittirIyam/sUtram/ApastambaH/shulbam/sarva-prastutiH/1/01.md

```diff
@@ -816,6 +828,38 @@ Ties are made at both ends of a cord of length equal to the given measure.
 Marks are given at its middle and at mid points of its two halves. After stretching the cord along the east-west line poles are fixed at the ties and the marks. With the two ties fixed at the two poles at the two outer marks (mid-points of two halves), the cord is stretched towards the south by the middle mark and a sign is given (at the point reached). With the two ties fixed at the middle pole, the cord is (again) stretched by its middle mark towards the south over the sign (previ- ously made) and a pole is fixed (at the point reached). With one tie (of the cord) fixed at this pole and the other tie at the eastern pole, the south- eastern corner is (now) obtained by (stretching the cord with) its middle mark. By removing the tie from the eastern pole and fixing it to the western pole, the south-western corner is likewise obtained by (stretching the cord with) the middle mark. In the same manner, the north-western and the north-eastern corners (are obtained).
 </details>
 
+<details><summary>English</summary>
+
+1.7. Construction of a square. This sūtra describes another method of construction of a square, not given by Baudhāyana. In Fig. 1, XY represents the given cord. T, U,V are marks at the middle of the cord XY, and of XT and TV respectively. EF is the east-west line. E, M, O, N, F are poles corresponding to X, U, T, V and Y. K represents the sign corresponding to T when the cord Xr is stretched after the ends X and Y are fixed at M and N. The mark S corresponding to T is 
+T 
+V 
```

Comment:
चित्राणि +उत्कृत्य रक्षणीयानि। विधिर् अत्रोक्तः - https://sanskrit.github.io/groups/dyuganga/projects/text/proofreading/editing/markdown/figures/

Reply:
चित्राणि कस्मात्  श्रीमन्

Reply:
pdf-तः। Fig. 1 इति दृश्यते खल्व् अत्र।

Reply:
https://archive.org/details/Kium_the-sulaba-sutras-of-baudayana-apastamba-katyayana-and-manava-by-sn-sen-and-ak-b/page/n7/mode/2up

Reply:
PDF न लब्धम्